### ISSUE TITLE
Newb Github Issue

### ISSUE THREAD

**Issue by urthling** [airsdk/apm#110]
For some reason I need to keep re-running: `. ~/.bashrc` - in every new window else, apm is not found.

Ok, main issue though, I keep getting the message:

`
There was an issue accessing the package, this is a private package so check you have a valid license and that you have correctly set your github access token
`

I set the token in my git profile, I updated my keychain -- is there somewhere in apm i'm missing to set something for git?

**Comment by marchbold** [comment]
Did you add the token to the apm configuration (https://github.com/airsdk/apm/wiki/Usage#authentication)

```
apm config set github_token ghp_XXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

**Comment by marchbold** [comment]
Did that resolve it? :)

**Comment by urthling** [comment]
Sure did! :)

**Comment by sweetnitro** [comment]
Sorry for unearthing this issue but the whole token situation seems to have evolved and I cannot access the private repositories.

The guide the wiki points to is obsolete, Personal access tokens have been removed and now there are either SSH tokens or GPG tokens.

I created a GPG token and tried setting it in APM, both with the token id or the subkey, with and without ghp_ prefix.. nothing works.

![image](https://github.com/airsdk/apm/assets/46676864/8104303a-e748-469a-a867-5e8f23779090)

Thanks for your help
